cmd/manager/main.go

```diff
@@ -2,23 +2,21 @@ package main
 
 import (
 	"flag"
+	"os"
 	"runtime"
 
 	"github.com/kontena/kubelet-rubber-stamp/pkg/apis"
 	"github.com/kontena/kubelet-rubber-stamp/pkg/controller"
-	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
-	sdkVersion "github.com/operator-framework/operator-sdk/version"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	"k8s.io/klog"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
-	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
 )
 
 func printVersion() {
 	klog.V(2).Infof("Go Version: %s", runtime.Version())
 	klog.V(2).Infof("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH)
-	klog.V(2).Infof("operator-sdk Version: %v", sdkVersion.Version)
```

Comment:
This info doesn't appear to be exposed anywhere, that I could tell

See https://github.com/operator-framework/operator-sdk/commits/master/internal/version/version.go (it's been progressively getting removed)